docs/crate-ownership.md

```diff
@@ -30,5 +32,5 @@ To promote healthy discussion, and ensure we work on impactful crates that cover
 To ensure further maintenance and publishing rights, team ownership should be added to each public crate.
 
 ```
-cargo owner -a github:unicode-org/teamname
+cargo owner -a github:icu4x/cargo-publish
```

Comment:
oh there's no actual icu4x github org, we should ask the unicode org admins to create a `unicode-org/icu4x-publish` team

Reply:
Yes. I'll keep what we have for now, since we are not going to publish on crates.io any time soon.

Reply:
Do you want to change this to `unicode-org/icu4x-sc/cargo-publish`?

Reply:
@sffc I don't think you can have two slashes, unless GitHub team names allow slashes. But something of that sort, once you get the team made

Reply:
@nciric I think keeping what we have is a bit dangerous since the icu4x organization doesn't exist and anyone can claim it and become an owner of a crate that had this command run on it.

We should remove this part or create a unicode org team, or perhaps just default add Zibi and Shane as owners for everything until we have that set up